packages/common/src/server.ts

```diff
@@ -155,9 +171,18 @@ export class YogaServer<
     TUserContext & TServerContext & YogaInitialContext
   >
   public logger: YogaLogger
-  private readonly corsOptionsFactory: (request: Request) => CORSOptions =
-    () => ({})
-  protected readonly graphiql: GraphiQLOptions | false
+  private readonly corsOptionsFactory: (
+    request: Request,
+    ...args: {} extends TServerContext
+      ? [serverContext?: TServerContext | undefined]
+      : [serverContext: TServerContext]
```

Comment:
It would be nice if we can refactor this type

Reply:
we just added it? :D

Reply:
What do you mean by refactoring?